src/createServer.js

```diff
@@ -1,3 +1,42 @@
+/* eslint-disable max-len */
 // Write code here
 // Also, you can create additional files in the src folder
 // and import (require) them here
+
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const { handleErrors } = require('./utils/handleErrors');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    res.setHeader('Content-Type', 'application/json');
+
+    const { pathname, searchParams } = new URL(
+      req.url,
+      `http://${req.headers.host}`,
+    );
+    const textToConvert = pathname.slice(1);
+    const targetCase = searchParams.get('toCase');
+
+    const hasAnyErrors = handleErrors(res, textToConvert, targetCase);
+
+    if (hasAnyErrors) {
+      return;
```

Comment:
i think it would be a better practice to use sendError here (also improving code readability